perfmetrics/scripts/load_tests/python/README.md

```diff
@@ -0,0 +1,146 @@
+# Python load testing tool
+
+Python load testing tool is to generate load on the machine it is run using a
+given task and then report the latencies of performing the task over the span
+of load test.
+A task is a piece of python code that is executed in multiple threads 
+in multiple processes. Users can define their own tasks, pass them to the tool 
+and pass different flags to configure the load testing.
+
+Example usage:
+```
+python3 load_test.py --tasks-python-file-path /path/to/task/module.py 
+--num-processes 32 --num-threads 2 --output-dir /dir/for/test/output 
+--run-time 60
+```
+In the above example usage, two threads are spawned in each of the 32 processes
+where each thread runs the task defined in /path/to/task/module.py in a
+continuous loop till 60 seconds. After 60 seconds, the results containing 
+latencies are saved in /dir/for/test/output.
+
+# Prerequisites
+
+* python3
+* python packages mentioned in [requirements.txt]
+
+[requirements.txt]: ./requirements.txt
+
+# Supported flags
+[load_test.py] is the script used that can be used to run load test. The script
+accepts many flags to customize load testing configuration. Below are some 
+important flags that can be passed to the script:
+* ```--tasks-python-file-path```: Path to python module (file) containing task 
+classes implementing task.LoadTestTask.
+* ```--tasks-yaml-file-path```: Path to yaml file containing configurations for 
+tasks. Note: Configurations in this file can only be of defined and recognised 
```

Comment:
We can also add the note, in case of when both tasks-python-file-path and tasks-yaml-file-path are passed to the testing-tool. We only process the tasks-python-file-path, yaml-file will be ignored.

Reply:
Okay, in this case we will get an error, while parsing the logs.